ricecooker/chefs.py

```diff
@@ -260,6 +260,20 @@ def get_channel(self, **kwargs):
             return channel
 
         elif hasattr(self, 'channel_info'):
+            # Before going any further, make sure there aren't template id values in their channel info.
+            template_domains = ['<yourdomain.org>']
+            using_template_domain = self.channel_info['CHANNEL_SOURCE_DOMAIN'] in template_domains
+            if using_template_domain:
+                config.LOGGER.error("Template source domain detected. Please set CHANNEL_SOURCE_DOMAIN before running this chef.")
+
+            template_ids = ['<unique id for the channel>', '<yourid>']
+            using_template_source_id = self.channel_info['CHANNEL_SOURCE_ID'] in template_ids
+            if using_template_source_id:
+                config.LOGGER.error("Template source ID deteected. Please set CHANNEL_SOURCE_ID before running this chef.")
```

Comment:
I was confused by the word "Template" on first reading wondering what template. Not sure if I have better alternative though...  Example maybe? Default values?

Also "set" is a bit confusing. Maybe "Change from the default". 

Leaving off perchance inspiration comes before tomorrow, but since none of my suggestions are any better than current options, I'm going to merge as-is tmrw AM.